docs/api.md

```diff
@@ -1,3 +1 @@
 # API Reference
-
```

Comment:
when i left it, it caused an error when I ran `mkdocs serve`:
`ERROR   -  Error reading page 'api.md':
ERROR   -  Could not collect 'blueprint.hello'`

 i'll try re-add the example as a string to avoid the error

Reply:
> ERROR   -  Error reading page 'api.md': ERROR   -  Could not collect 'blueprint.hello'

Right, I think it might be because the project itself was not installed. 
You need to:

```
pip install -e .
```

From the root so mkdocs can import the `blueprint` module.

Reply:
There are some changes needed in order to make the workflows work on this repo, I will open a PR

Reply:
@daavoo - okay do you want me to hold-off merging this? or can i go ahead and then you will do another PR?

Reply:
Is ok to merge this, I will open a different PR